scripts/configure.mjs

```diff
@@ -651,7 +651,7 @@ export function configure(params, fs = nodefs) {
     if (!fs.existsSync(packageManifest)) {
       // We cannot assume that the app itself is an npm package. Some libraries
       // have an 'example' folder inside the package.
-      console.warn(`package.json not found in path "${packagePath}". Skipped applying modifications for react-native-test-app.`);
+      warn(`'package.json' was not found in path '${packagePath}'; skipped modifications`);
```

Comment:
On second thought, this sounds like we're skipping ALL modifications.
```suggestion
      warn(`skipped modifying 'package.json' because it was not found in path '${packagePath}'`);
```